### PR TITLE
DTGB-600: Added layer of js and sass to fix jQuery UI range slider facet + accessibility fixes.

### PR DESCRIPTION
**Pull request by meirege**
…cet.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: templates/custom/nodes/node--outdoor-location--teaser.html.twig [hunk 1]
```diff
@@ -84,7 +84,7 @@
 
 {{ attach_library('classy/node') }}
 
-<article{{ attributes.addClass(classes) }}>
+<article{{ attributes.addClass(classes).removeAttribute('role') }}>
```
Comment (delrueba) [inline templates/custom/nodes/node--outdoor-location--teaser.html.twig:87]: Ooh, ladida :+1:

File: templates/custom/nodes/node--outdoor-location--teaser.html.twig [hunk 2]
```diff
@@ -84,7 +84,7 @@
```
Comment (delrueba) [inline templates/custom/nodes/node--outdoor-location--teaser.html.twig:84]: Isn't this the teaser that doesn't have alt attributes too?

File: templates/custom/nodes/node--outdoor-location--teaser.html.twig (hunk 1 above)
Comment (meirege) [reply templates/custom/nodes/node--outdoor-location--teaser.html.twig:87]: 👍

File: gent_base.theme [hunk 3]
```diff
@@ -783,6 +783,14 @@ function gent_base_preprocess_pager(&$variables) {
   $variables['items']['last']['text'] = $total;
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function gent_base_preprocess_facets_item_list(&$variables) {
+  $test = $variables;
+  $variables['testje'] = 'items';
```
Comment (hstandaert) [inline gent_base.theme]: Is this supposed to be in there? 😁

File: templates/custom/nodes/node--outdoor-location--teaser.html.twig (hunk 2 above)
Comment (meirege) [reply templates/custom/nodes/node--outdoor-location--teaser.html.twig:84]: Not for this PR :)

Comment (meirege) [reply templates/custom/nodes/node--outdoor-location--teaser.html.twig:84]: Not for this PR :)

Comment (meirege) [reply templates/custom/nodes/node--outdoor-location--teaser.html.twig:84]: Not in this PR. :)

**Review by hstandaert — APPROVED** [review]

**Pull request merged**